packages/client/modules/email/components/NotificationSummaryEmail.tsx

```diff
@@ -33,23 +35,41 @@ export interface NotificationSummaryProps {
   appOrigin: string
   preferredName: string
   notificationCount: number
+  notificationRefs: EmailNotificationPicker_notification$key[]
 }
+
 export default function NotificationSummaryEmail(props: NotificationSummaryProps) {
-  const {appOrigin, notificationCount, preferredName} = props
+  const {appOrigin, notificationCount, preferredName, notificationRefs} = props
+  const additionalNotifications = notificationCount - notificationRefs.length
   const tasksURL = makeAppURL(appOrigin, 'me/tasks', {searchParams: notificationSummaryUrlParams})
   return (
     <Layout maxWidth={544}>
       <EmailBlock innerMaxWidth={innerMaxWidth}>
-        <Header appOrigin={appOrigin} corsOptions={EMAIL_CORS_OPTIONS} />
-        <p style={copyStyle}>{`Hi ${preferredName} -`}</p>
-        <p style={copyStyle}>
+        <Header align='center' appOrigin={appOrigin} corsOptions={EMAIL_CORS_OPTIONS} />
+        <p
+          style={{...copyStyle, textAlign: 'center', marginBottom: '0px'}}
+        >{`Hi ${preferredName} 👋`}</p>
+        <p style={{...copyStyle, textAlign: 'center'}}>
           {'You have '}
           <span style={{fontWeight: 600}}>
-            {`${notificationCount} new unread ${plural(notificationCount, 'notification')}`}
+            {`${notificationCount} new ${plural(notificationCount, 'notification')}`}
           </span>
-          {' — see what’s changed with your teams.'}
+          {' from Parabol.'}
         </p>
-        <Button url={tasksURL}>{'See My Dashboard'}</Button>
+        {notificationRefs.map((notificationRef, i) => {
+          return (
+            <EmailNotificationPicker
+              key={i}
+              appOrigin={appOrigin}
+              notificationRef={notificationRef}
+            />
+          )
+        })}
+        <Button url={tasksURL}>
+          {additionalNotifications > 0
```

Comment:
-1 this should be `{additionalNotifications > 0 && notificationRefs.length > 0`, otherwise we will write text as "You have 1 new notification from Parabol. ...and 1 more" if there is only 1 unsupported notification.